lemur/plugins/lemur_aws/plugin.py

```diff
@@ -577,7 +577,9 @@ class S3DestinationPlugin(ExportDestinationPlugin):
             "name": "prefix",
             "type": "str",
             "required": False,
+            "validation": check_validation("[^/].*"),
             "helpMessage": "Must be a valid S3 object prefix!",
+            "default": "lemur"
```

Comment:
do we want to set the default prefix empty?

Reply:
+1, empty is potentially safer and may align with current behavior, if code could handle None path anywhere.

Reply:
sure thing, the validation regex gets a bit more complex, but it's nothing too unmanageable